addons/point_of_sale/static/src/js/models.js

```diff
@@ -1782,7 +1781,7 @@ exports.Order = Backbone.Model.extend({
     },
 
     initialize_validation_date: function () {
-        this.validation_date = this.validation_date || new Date();
+        this.validation_date = new Date();
```

Comment:
Hmm, I think that this single one-line change should achieve what we want. In this case whenever initialize_validation_date gets called (only happens once in the code, when validating the order) it will simply set the current time. I don't remember why I did 

```
this.validation_date || new Date();
```

But let's assume I was confused?